app/containers/EliteApiLoader/sagas.js

```diff
@@ -201,12 +202,20 @@ export function* loadCaseNoteTypesSubTypes() {
   const apiServer = yield select(selectApi())
 
   yield call(preloadAllCaseNoteTypesSubTypes, apiServer)
+}
+
+export function* loadMyCaseNoteTypesSubTypes() {
+  const apiServer = yield select(selectApi())
+
   yield call(preloadUserCaseNoteTypes, apiServer)
```

Comment:
separate functions now load either the reference data case notes or the user's case notes